platform/ui/src/components/SidePanel/SidePanel.tsx

```diff
@@ -67,15 +68,22 @@ const position = {
 };
 
 const SidePanel = ({
+  servicesManager,
   side,
   className,
   activeTabIndex: activeTabIndexProp,
   tabs,
+  openWhenPanelActivated,
 }) => {
+  const panelService: PanelService = servicesManager.services.panelService;
+
   const { t } = useTranslation('SidePanel');
 
+  const [hasBeenOpened, setHasBeenOpened] = useState(
```

Comment:
please add a comment above that this tracks it has been opened from the begining of the app

Reply:
actually it does not track from the beginning of the app, only for the life of the SidePanel. So if one returns to the Search page, this will get reset.

Reply:
if so please add that comment there